app/templates/services/service.html

```diff
@@ -86,8 +88,15 @@
   {% endif %}
 {% endblock %}
 
+{% block summary_row %}
+  {% if not question.is_empty %}
+    {{ summary.field_name(question.label) }}
+    {{ summary[question.type](question.value, question.assurance) }}
+  {% endif %}
+{% endblock %}
+
 {% block after_sections %}
-  {% if service_data.status == 'published' and not remove_requested %}
+  {% if service_data.status == 'published' and not remove_requested and "digital-outcomes-and-specialists" != service_data.frameworkFramework %}
```

Comment:
Another one where I'd have put the things the other way round, that way it would match the `service_data.status == 'published'` condition.